docs/source/overview.md

```diff
@@ -25,19 +25,58 @@ flowchart LR
 These measurement data are the focus of the data source content which follows.
 
 ### Data Source Locations
 
 ```{eval-rst}
 Data sources may be provided to CytoTable using local filepaths or remote object-storage filepaths (for example, AWS S3, GCP Cloud Storage, Azure Storage).
 We use `cloudpathlib <https://cloudpathlib.drivendata.org/~latest/>`_  under the hood to reference files in a unified way, whether they're local or remote.
+```
+
+#### Data Source Locations in the Cloud
+
+Cloud-based data sources used through CytoTable leverage [cloudpathlib](https://cloudpathlib.drivendata.org/~latest/).
+Examples of paths to these sources leverage specialized prefixes and may be found below.
+
+- [Amazon S3](https://en.wikipedia.org/wiki/Amazon_S3): `s3://bucket_name/object_name`
+- [Google Cloud Storage](https://en.wikipedia.org/wiki/Google_Cloud_Storage): `gc://bucket_name/object_name`
+- [Azure Blob Storage](https://en.wikipedia.org/wiki/Microsoft_Azure#Storage_services): `az://container_name/blob_name`
+
+##### Cloud Service Configuration and Authentication
 
+```{eval-rst}
 Remote object storage paths which require authentication or other specialized configuration may use cloudpathlib client arguments (`S3Client <https://cloudpathlib.drivendata.org/~latest/api-reference/s3client/>`_, `AzureBlobClient <https://cloudpathlib.drivendata.org/~latest/api-reference/azblobclient/>`_, `GSClient <https://cloudpathlib.drivendata.org/~latest/api-reference/gsclient/>`_) and :code:`convert(..., **kwargs)` (:mod:`convert() <cytotable.convert.convert>`).
+
 For example, remote AWS S3 paths which are public-facing and do not require authentication (like, or similar to, :code:`aws s3 ... --no-sign-request`) may be used via :code:`convert(..., no_sign_request=True)` (:mod:`convert() <cytotable.convert.convert>`).
 ```
 
+Each cloud service provider may have different requirements for authentication (there is no fully unified API for these).
+Please see the [cloudpathlib](https://cloudpathlib.drivendata.org/~latest/) client documentation for more information on which arguments may be used for configuration with specific cloud providers (for example, [`S3Client`](https://cloudpathlib.drivendata.org/stable/api-reference/s3client/), [`GSClient`](https://cloudpathlib.drivendata.org/stable/api-reference/gsclient/), or [`AzureBlobClient`](https://cloudpathlib.drivendata.org/stable/api-reference/azblobclient/)).
+
+##### Cloud Service File Type Parsing Differences
+
+Data sources retrieved from cloud services are not all treated the same due to technical constraints.
+See below for a description of how each file type is treated for a better understanding of expectations.
+
+__Comma-separated values (.csv)__:
+
+CSV files stored on cloud services are read directly from the cloud service.
+
+__SQLite Databases (.sqlite)__:
+
+SQLite databases stored on cloud services are downloaded locally before other CytoTable work is performed.
```

Comment:
```suggestion
CytoTable downloads cloud-based SQLite databases locally before other CytoTable processing.
```

Reply:
Liking the project-subject wording a lot better, more clear from the audience-perspective, thanks!